db/migrations/mysql/000076_upgrade_lastrootpostat.up.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE Channels ALTER COLUMN LastRootPostAt SET DEFAULT 0;
```

Comment:
Can we guard this with a if statement? Like we do for others, it's about being defensive.

Reply:
It's idempotent anyways right?